pkgs/development/tools/misc/indent/default.nix

```
@@ -10,14 +10,17 @@ stdenv.mkDerivation rec {
   };
 
   patches = [ ./darwin.patch ];
+  makeFlags = [ "AR=${stdenv.cc.targetPrefix}ar" ];
 
-  buildInputs = [ texinfo ];
+  nativeBuildInputs = [ texinfo ];
+  pkgsBuildBuild = [ buildPackages.stdenv.cc ];
```

Realizing that someone might not recognize what this does and remove it since it's not needed in the native case, might be good to leave a remark here like
```suggestion
  pkgsBuildBuild = [ buildPackages.stdenv.cc ]; # needed when cross-compiling
```